docs/alerts/monitors/monitor-faq.md

```diff
@@ -48,6 +48,12 @@ deployment=acme region=us-east1 metric=container_cpu_usage_seconds_total | rate
 
 And so on.
 
+## Is there a limit to the number of monitors I can have?
+
+You can have up to 5,000 monitors (active and inactive) simultaneously.
```

Comment:
By default the limit is per the subscription type. user need to reach out to customer support to bump to 5k monitors. i think we should mention that.

Reply:
Thanks for the quick feedback! Where can the user find their current subscription type limit? I don't see it under **Account** > **Administration**.

Reply:
documented @ https://docs.google.com/document/d/18GOB3Ppo7wa1ft-RY1ePk1ymgT0x43AuAy1rhG3Nf6k/edit# but don't know if we want to do @ docs

Reply:
@kimsauce , 
can we say , you can increase the monitor limit up to 5000 by reaching out to customer support?